mappings/net/minecraft/network/encryption/ChatSigner.mapping

```diff
@@ -0,0 +1,9 @@
+CLASS net/minecraft/class_7470 net/minecraft/network/encryption/ChatSigner
```

Comment:
Recommended rename to `MessageSigner`, same rationale

Reply:
Downvoting, this is not a general purpose signer I believe

Reply:
Then I'd recommend `ChatMessage` signer. So we should use `ChatMessage` for chat messages (or we can just call them `Chat`, without message suffix), and use `Message` as the super type of chat messages and other system messages etc. (which mojang calls "chat")

Reply:
Sounds good, hope i can rename it properly on phone